README_RECON.md

```diff
@@ -0,0 +1,165 @@
+   databases.
+
+2. **Install the JAR file on Databricks:**
+   Upon completing the download, proceed to install the JAR file onto your Databricks cluster. Refer
+   to [this page](https://docs.databricks.com/en/libraries/workspace-libraries.html#upload-a-jar-python-egg-or-python-wheel)
+   for comprehensive instructions on how to upload a JAR file, Python egg, or Python wheel to your Databricks workspace.
+
+3. This installation is a necessary step to enable seamless comparison between Oracle and Databricks, ensuring that the
+   required Oracle JDBC functionality is readily available within the Databricks environment.
+
+## Commonly Used Custom Transformations
+
+| source_type | data_type     | source_transformation                                                | target_transformation                           | source_value_example | target_value_example | comments                                                                                    |
+|-------------|---------------|----------------------------------------------------------------------|-------------------------------------------------|----------------------|----------------------|---------------------------------------------------------------------------------------------|
+| oracle      | number(10,5)  | trim(to_char(coalesce(<col_name>,0.0), '99990.99999'))               | cast(coalesce(<col_name>,0.0) as decimal(10,5)) | 1.00                 | 1.00000              | this can be used for any precision and scale by adjusting accordingly in the transformation |
+| snowflake   | array         | array_to_string(array_compact(<col_name>),',')                       | concat_ws(',', <col_name>)                      |                      |                      | in case of removing "undefined" during migration(converts sparse array to dense array)      |
+| snowflake   | array         | array_to_string(array_sort(array_compact(<col_name>),true,true),',') | concat_ws(',', <col_name>)                      |                      |                      | in case of removing "undefined" during migration and want to sort the array                 |
+| snowflake   | timestamp_ntz | date_part(epoch_second,<col_name>)                                   | unix_timestamp(<col_name>)                      |                      |                      | convert timestamp_ntz to epoch for getting a match b/w snowflake and databricks             | 
+
+## Data Reconciliation Examples:
+
+For more Reconciliation Config examples ,please refer to [sample_notebook][link].
+
+[link]: ./docs/reconciliation-configs-examples.py
+
+## Note:
+**In Reconciliation** 
+1. Databricks with UC is the only target
+2. Duplicates are not covered
+3. User Transformations are not applied for Schema Validation.Only select_columns,drop_columns and column_mapping is valid for schema  
+validation
+4. Aggregate transformations or multi-column transformations are not supported
+5. Not all complex data types are supported
```

Comment:
please add details on how various `report type` behaves under this section.

Reply:
Added at the top level